pages/admin/createcohort.js

```diff
@@ -28,7 +28,7 @@ export default function CreateCohort() {
     return toast.success('Cohort criado com sucesso!')
   }
   return (
-    <Layout>
```

Comment:
Is it necessary to leave this empty HTML tag here? `<>` and all other pages?

Reply:
https://trello.com/c/7DFVnptc/193-refatora%C3%A7%C3%A3o-navbar-footer

resolved